File: PY-01/2/while.py
jumlah_nilai = int(input("Masukkan jumlah nilai: "))
i = 0

total = 0
while i < jumlah_nilai:
    nilai = int(input("Nilai %d: " % (i+ 1)))
    total += nilai
    i += 1
    
total /= jumlah_nilai

print("Rata-rata nilai siswa: %f" % total)

File: CPP-02/Session 5/while.cpp
#include<iostream>

using namespace std;

int main () {
    int jumlahNilai;
    cin >> jumlahNilai;
    int total = 0;
    int i = 0;
    while (i < jumlahNilai) {
        int nilai;
        cout << "nilai ke-" << i+1 << ": ";
        cin >> nilai;
        total += nilai;
        i++;
    }
    cout << "rata rata" << jumlahNilai << "nilai adalah" << total/jumlahNilai << endl;
    return 0;

}